### PR TITLE
airbnb -> mulesoft

### PR DESCRIPTION
**Pull request by yianL**

### CODE REVIEW

File: packages/eslint-config-mulesoft-base/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Lints ES5 and below. Requires `eslint` and `eslint-plugin-import`.
 
 2. Add `"extends": "airbnb-base/legacy"` to your .eslintrc
```
Comment (nescalante) [inline packages/eslint-config-mulesoft-base/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft-base/README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Lints ES5 and below. Requires `eslint` and `eslint-plugin-import`.
...
 
-See [Airbnb's overarching ESLint config](https://npmjs.com/eslint-config-airbnb), [Airbnb's Javascript styleguide](https://github.com/airbnb/javascript), and the [ESlint config docs](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) for more information.
+See [Airbnb's overarching ESLint config](https://npmjs.com/eslint-config-mulesoft), [Airbnb's Javascript styleguide](https://github.com/airbnb/javascript), and the [ESlint config docs](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) for more information.
```
Comment (nescalante) [inline packages/eslint-config-mulesoft-base/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 3]
```diff
@@ -30,11 +30,11 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+ and
 
 ### eslint-config-airbnb/base
 
-This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-mulesoft-base).
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 4]
```diff
@@ -30,11 +30,11 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+ and
...
 
 ### eslint-config-airbnb/legacy
 
-This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-mulesoft-base).
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 5]
```diff
@@ -30,11 +30,11 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+ and
 
 ### eslint-config-airbnb/base
 
-This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-mulesoft-base).
 
 ### eslint-config-airbnb/legacy
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 6]
```diff
@@ -30,11 +30,11 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+ and
 
 ### eslint-config-airbnb/base
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # eslint-config-airbnb
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/README.md [hunk 8]
```diff
@@ -30,11 +30,11 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+ and
...
 
-This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-mulesoft-base).
 
 ### eslint-config-airbnb/legacy
 
-This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
+This entry point is deprecated. See [eslint-config-airbnb-base](https://npmjs.com/eslint-config-mulesoft-base).
 
 See [Airbnb's Javascript styleguide](https://github.com/airbnb/javascript) and
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/README.md]: replace airbnb here

File: packages/eslint-config-mulesoft/package.json [hunk 9]
```diff
@@ -1,19 +1,19 @@
 {
-  "name": "eslint-config-airbnb",
-  "version": "10.0.1",
+  "name": "eslint-config-mulesoft",
+  "version": "0.0.1",
   "description": "Airbnb's ESLint config, following our styleguide",
```
Comment (nescalante) [inline packages/eslint-config-mulesoft/package.json]: replace airbnb here

File: packages/eslint-config-mulesoft-base/package.json [hunk 10]
```diff
@@ -1,19 +1,19 @@
 {
-  "name": "eslint-config-airbnb-base",
-  "version": "5.0.3",
+  "name": "eslint-config-mulesoft-base",
+  "version": "0.0.1",
   "description": "Airbnb's base JS ESLint config, following our styleguide",
```
Comment (nescalante) [inline packages/eslint-config-mulesoft-base/package.json]: replace airbnb here

**Comment by yianL** [comment]
Fixed all the comments

**Pull request merged**